### PR TITLE
Use murmur32 hashing algorithm

### PR DESCRIPTION
**Pull request by maksimf**

### CODE REVIEW

**Comment by maksimf** [comment]
@patricioe This PR can be merged

**Comment by maksimf** [comment]
@patricioe Added specs for both alpha and non-alpha keys, all green

File: tasks/benchmark_hashing_algorithm.rake [hunk 1]
```diff
@@ -0,0 +1,51 @@
+require 'benchmark'
+require 'digest/murmurhash'
+
+desc 'Benchmark murmur32 hashing algorithm'
```
Comment (patricioe) [inline tasks/benchmark_hashing_algorithm.rake]: @maksimf how do I run this? Are there instructions anywhere?

File: lib/splitclient-rb/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module SplitIoClient
-  VERSION = '3.1.4'
+  VERSION = '3.2.0'
```
Comment (patricioe) [inline lib/splitclient-rb/version.rb]: make it 4.0.0 please.

File: tasks/benchmark_hashing_algorithm.rake (hunk 1 above)
Comment (maksimf) [inline tasks/benchmark_hashing_algorithm.rake]: `bundle exec rake benchmark_hashing_algorithm`

**Comment by maksimf** [comment]
@patricioe Done

**Comment by maksimf** [comment]
@patricioe I've added the ability to use both legacy and murmur hash

**Comment by maksimf** [comment]
@patricioe Don't merge it yet, I need to add legacy hash support

**Comment by maksimf** [comment]
@patricioe ready for review/merge

**Pull request merged**